pkg/document/generate.go

```diff
@@ -77,6 +78,7 @@ func applyMarkDownFormat(output bytes.Buffer) bytes.Buffer {
 
 	re = regexp.MustCompile(`\n{3,}`)
 	outputString = re.ReplaceAllString(outputString, "\n\n")
```

Comment:
You could just remove one line break that it works as you intended.

```suggestion
	outputString = re.ReplaceAllString(outputString, "\n")
```

Reply:
This logic is replacing all triple or more newlines to two. With your change, we reduce it to 1. Does this make sense? Or is it better just to clean the double newlines at the EOF?

Reply:
Make the change and see the test charts. In principle it doesn't affect that much. 

But you are right, it could break and the way to less affect it is by using the strings as you did.

Reply:
I will just resolve this and accept as I see why you did it and there is no reason for such a lengthy discussion over formatting lines.